#!/usr/bin/env python
PACKAGE = "gen3_control"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("vel_lim_coeff", double_t, 0, "Scale factor", 25, 1, 100)
gen.add("acc_lim_coeff", double_t, 0, "Scale factor", 25, 1, 100)

exit(gen.generate(PACKAGE, "gen3_control", "DynVar"))
